src/Components/Display.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const display = (props) => <div className="Display"> { props.result } </div>;
+
+display.propTypes = { result: PropTypes.number };
+
+export default display;
```

Comment:
- [ ] As it said on the instructions, we need to define a default value as `'0'` for the `result` prop. I know you are passing already in the App component the prop with a default value, but the best way to do that is defining it here, in the Display component. This way you will inform the Proptypes config object that the result object is not always required and have a default value. You will notice that Eslint will start complaining (after been properly set) about result prop was not declare as required in the display component propTypes object and does have a default value.
You can specify default value for props this way:
```
display.defaultProps = {
  result: '0'
}
```